.github/workflows/bumpdevversion.yml

```diff
@@ -0,0 +1,115 @@
+
+          ## Step 2b. Build new version string (e.g. 6.2.0.dev1+hijklmn)
+
+          # Step 2b i. Get current tag of latest dev build (e.g. 6.2.0)
+          export TAG=$(echo $OLD_VERSION_STRING | grep -o -P '.*(?=.dev)')
+
+          # Step 2b ii. Add 1 to the previous dev version number (e.g. 6.2.0.dev0+abcdefg would output 1, i.e. for dev1)
+          export DEV_VERSION_NUM=`expr 1 + $(yq read recipe/meta.yaml package.version | grep -o -P '(?<=dev).*(?=\+)')`
+
+          # Step 2b iii. Get shortened new sha hash value (e.g. hijklmn)
+          export SHORT_HASH=$(echo $NEW_SHA_HASH | cut -c 1-7)
+
+          # Step 3d. Finally build the new version string (e.g. 6.2.0.dev1+hijklmn)
+          echo ::set-output name=NEW_VERSION_STRING::$TAG.dev$DEV_VERSION_NUM+$SHORT_HASH
+
+      # Change development package version and source git revision
+      - name: Update package version and git revision
+        run: |
+          # Update the recipe/meta.yaml file with new version info
+          yq write --inplace recipe/meta.yaml package.version ${{ steps.version_info.outputs.NEW_VERSION_STRING }}
+          yq write --inplace recipe/meta.yaml source.git_rev ${{ steps.version_info.outputs.NEW_SHA_HASH }}
+
+          # Fix incorrectly formatted {{ compiler('c') }}
+          yq write --inplace recipe/meta.yaml requirements.build[1] "{{ compiler('c') }}"
+          sed --in-place "s/'{{ compiler(''c'') }}'/{{ compiler('c') }}/g" recipe/meta.yaml
+
+          # Ensure <two spaces>#<one space>[<expression>] form
+          sed --in-place "s/\ \#\ \[/  # [/g" recipe/meta.yaml
+
+      # Take a look at recipe/meta.yaml contents
+      - name: Check meta.yaml contents
+        run: cat recipe/meta.yaml
+
+      # Open a Pull Request from personal fork to the upstream repo
+      # The GMT_FEEDSTOCK_TOKEN personal access token secret should have
+      # a scope of "repo: public_repo" for access to public repositories
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3.4.0
+        with:
+          title: "Bump GMT from ${{ steps.version_info.outputs.OLD_VERSION_STRING }} to ${{ steps.version_info.outputs.NEW_VERSION_STRING }}"
+          commit-message: "Bumps GMT dev version from ${{ steps.version_info.outputs.OLD_VERSION_STRING }} to ${{ steps.version_info.outputs.NEW_VERSION_STRING }}"
+          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          # reviewers: "weiji14"  # TODO change to maintainer on conda-forge/gmt-feedstock
+          base: devel
+          branch: update/${{ steps.version_info.outputs.NEW_VERSION_STRING }}
+          delete-branch: true
+          token: ${{ secrets.GMT_FEEDSTOCK_TOKEN }}
+          draft: true
+          push-to-fork: GenericMappingTools/gmt-feedstock
+          body: |
+            It is very likely that the current package version for this feedstock is out of date.
```

Comment:
```suggestion
            *This is an automated pull request!*
            
            It is very likely that the current package version for this feedstock is out of date.
```

Reply:
I'll commit this change a bit later, otherwise it will trigger another automated PR.